### PR TITLE
[SIMD.js] Implement SIMD.float64x2.swizzle and SIMD.float64x2.shuffle ap...

### PR DESCRIPTION
**Pull request by chuan9**
...is in runtime.

add testcases.

BUG=XWALK-3517

### CODE REVIEW

File: src/runtime/runtime-simd.cc [hunk 1]
```diff
@@ -513,6 +513,31 @@ RUNTIME_FUNCTION(Runtime_##TYPE##Swizzle) {                   \
 SIMD128_SWIZZLE_FUNCTIONS(DECLARE_SIMD_SWIZZLE_FUNCTION)
 
 
+#define SIMD128_SWIZZLE_64x2_FUNCTIONS(V)                     \
```
Comment (huningxin) [inline src/runtime/runtime-simd.cc]: If there is only one instance, why use template?

Comment (chuan9) [inline src/runtime/runtime-simd.cc]: @huningxin Just for style consistency, if you think we don't need to do this, I will convert the template to instance.

File: src/simd128.js [hunk 2]
```diff
@@ -249,6 +249,10 @@ FUNCTION(Float32x4)
 FUNCTION(Int32x4)
 endmacro
 
+macro SIMD128_SHUFFLE_64x2_FUNCTIONS(FUNCTION)
```
Comment (huningxin) [inline src/simd128.js]: Same as C++ implementation, no need to use macro here.

File: src/simd128.js [hunk 3]
```diff
@@ -292,6 +296,31 @@ function TYPEFUNCTIONJS(a4, b4) {
 }
 endmacro
 
+macro DECLARE_SIMD_SHUFFLE_64x2_FUNCTION(TYPE)
+function TYPESwizzleJS(x4, x, y) {
```
Comment (huningxin) [inline src/simd128.js]: `x4` is confusing. It is not a x4 type like float32x4.

File: src/simd128.js [hunk 4]
```diff
@@ -292,6 +296,31 @@ function TYPEFUNCTIONJS(a4, b4) {
...
+  var x_ = TO_INT32(x);
+  var y_ = TO_INT32(y);
+  if ((x_ < 0) || (x_ > 1) ||
+      (y_ < 0) || (y_ > 1)) {
+    throw MakeRangeError("invalid_simd_shuffle_lane_index");
+  }
+  return %TYPESwizzle(x4, x_, y_);
+}
+
+function TYPEShuffleJS(x4, y4, x, y) {
```
Comment (huningxin) [inline src/simd128.js]: ditto

**Comment by huningxin** [comment]
LGTM. Thanks the PR! I am going to merge it.

**Pull request merged**